cluster-scope/overlays/nerc-ocp-infra/configmaps/kustomization.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+commonLabels:
+  nerc.mghpcc.org/kustomized: "true"
```

Comment:
You don't need to set this here -- it's set by the top level `kustomization.yaml`.

Reply:
also done